docs/docs/configuration/object_detectors.md

```diff
@@ -1072,3 +1073,98 @@ wget -O yolov9-t.pt "https://github.com/WongKinYiu/yolov9/releases/download/v0.1
 sed -i "s/ckpt = torch.load(attempt_download(w), map_location='cpu')/ckpt = torch.load(attempt_download(w), map_location='cpu', weights_only=False)/g" ./models/experimental.py
 bin/python3 export.py --weights ./yolov9-t.pt --imgsz 320 --simplify --include onnx
 ```
+
+## DeGirum
```

Comment:
This needs to be moved up slightly, to below the RKNN detector (above downloading models section)